smartcontractkit/external-adapters-js
# Chore/Add DEBUG and NODE_ENV warnings

## Closes SC-56137

## Description

Log warning on startup when `DEBUG=true` and/or `NODE_ENV=development`.

## Changes

- Added DEBUG=true warning
- Added NODE_ENV=development warning

## Steps to Test

1. Set DEBUG to true
2. Set NODE_ENV to development
3. Start any adapter
4. Verify the warning logs for these env vars are present

## Quality Assurance
